client/src/components/BaseModal.vue

```diff
@@ -14,14 +14,16 @@
           <p>{{ modalBody }}</p>
         </div>
         <div class="modal-footer">
+          <BaseError :error-message="errorMessage" />
+
           <button
             type="button"
             class="btn"
             :class="isDangerousAction ? 'btn-danger' : 'btn-primary'"
-            data-bs-dismiss="modal"
```

Comment:
What are the pros and cons of removing data-bs-dismiss attribute compared to an extra line in every component which uses BaseModal?

Reply:
Wenn das Modal sich selber schließen kann, kann man nicht abwarten ob die Aktion erfolgreich war um dann eine Fehlermeldung anzuzeigen.